cmd/glasskube/cmd/install.go

```diff
@@ -200,9 +204,36 @@ var installCmd = &cobra.Command{
 				cliutils.ExitWithError()
 			}
 		}
+		if installCmdOptions.OutputOptions.Output != "" {
+			output, err := formatOutput(pkg, installCmdOptions.OutputOptions.Output)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "❗ Error: %v\n", err)
+				cliutils.ExitWithError()
+			}
+			fmt.Println(output)
+		}
 	},
 }
 
+func formatOutput(pkg *v1alpha1.Package, format OutputFormat) (string, error) {
```

Comment:
could you please make sure here to add the `GroupVersionKind` to the `pkg` if missing? Like here: https://github.com/glasskube/glasskube/pull/670/commits/18e921c8911ef438cac964d9b07254a3c8a3d68d